packages/e2e/tsconfig.json

```diff
@@ -1,12 +1,14 @@
 {
-    "compilerOptions": {
-        "module": "commonjs",
-        "target": "es5",
-        "noImplicitAny": false,
-        "sourceMap": false,
-        "lib": ["es2017"],
-        "types": [
-            "@sakuli/legacy-types"
-        ]
-    }
-}
\ No newline at end of file
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "ES2018",
```

Comment:
I assume ES2018 is possible, because we don't rely on browser engines but on node 12, right?